eslzArm/managementGroupTemplates/policyAssignments/DINE-SQLAuditingPolicyAssignment.json

```diff
@@ -61,5 +72,10 @@
             }
         }
```

Comment:
Shouldnt we replace Owner here with the Log Analytics Contributor & SQL Security Manager finer grained roles? I see you did it for the management sub, but not for the Landing Zones MG where the policy gets assigned.

Reply:
you are right - I will see to it.